distributed/comm/tests/test_ucx.py

```diff
@@ -276,30 +276,22 @@ async def test_ucx_localcluster(processes, cleanup):
 
 
 @pytest.mark.slow
-@pytest.mark.asyncio
-async def test_stress(cleanup):
+@gen_cluster(client=True, scheduler_kwargs={"protocol": "ucx"}, timeout=240)
```

Comment:
It looks like we've dropped `host=HOST` which we were previously using in the `LocalCluster` setup

Reply:
reverted